Mathlib/Order/IsNormal.lean

```diff
@@ -28,26 +28,37 @@ variable {α β γ : Type*} {a b : α} {f : α → β} {g : β → γ}
 namespace Order
 
 /-- A normal function between well-orders is a strictly monotonic continuous function. -/
+@[mk_iff isNormal_iff']
 structure IsNormal [LinearOrder α] [LinearOrder β] (f : α → β) : Prop where
   strictMono : StrictMono f
-  isLUB_image_Iio_of_isSuccLimit {a : α} (ha : IsSuccLimit a) : IsLUB (f '' Iio a) (f a)
+  /-- This condition is the RHS of the `IsLUB (f '' Iio a) (f a)` predicate. -/
```

Comment:
```suggestion
  /-- This condition is the RHS of the `IsLUB (f '' Iio a) (f a)` predicate, which is sufficient
  since the LHS is implied by ... -/
```
where I've left the last clause for you